lib/src/test/kotlin/graphql/nadel/validation/NadelTypeValidationTest.kt

```diff
@@ -474,50 +474,6 @@ class NadelTypeValidationTest : DescribeSpec({
             assert(error.overallField.name == "nextgenSpecific")
         }
 
-        it("ignores extension union members declared by other services") {
```

Comment:
Can we add tests for the checks performed by `NadelUnionValidation`?

Reply:
Yeah my bad, got distracted by E2E tests and forgot. Will add some.

Reply:
Added

Reply:
Good stuff!!! Thanks for that.